tests/lisa/test_trace.py

```diff
@@ -21,6 +21,23 @@
 
 from trace import Trace
 
+
+#
+# Helpers for generating example text-format trace events.
+# Doesn't have the exact same whitespace as real trace output, but that
+# shouldn't matter.
+#
+
+def _event_common(_comm, _pid, _cpu, timestamp):
+    return "{_comm}-{_pid}  [{_cpu:0>3}] {timestamp}: ".format(**locals())
+
+def sched_switch(timestamp, cpu,
+                 prev_comm, prev_pid, prev_state, next_comm, next_pid):
+    return (_event_common(prev_comm, prev_pid, cpu, timestamp) + "sched_switch: "
+            "prev_comm={prev_comm} prev_pid={prev_pid} prev_prio=120 prev_state={prev_state} "
```

Comment:
Shouldn't `prev_prio` and `next_prio` be parameters too?

Reply:
I didn't include it because it doesn't make any difference to the code under test. If it became necessary we could always add it earlier.

The main question about this code is.. do you think the use of this `sched_switch` helper function actually makes the code easier or harder to read? I cannot decide.

Reply:
Since we are talking about tests, it's probably a lot more clear to just have the string with the events in the test itself. I think it makes it much easier to read the test, otherwise one would have to go through the helpers to see what they produce.

Reply:
OK, you're probably right - this is over-complicating things. I'll revert to just having the traces as text.

Reply:
Can you also merge together the last two? They have the same commit message and they modify the same file.